src/js/components/Select/SelectContainer.js

```diff
@@ -342,9 +352,37 @@ class SelectContainer extends Component {
     return result;
   };
 
+  renderClear = () => {
+    const { clear, theme } = this.props;
+    const { renderValue, label } = clear;
+    if (renderValue) {
+      return renderValue({ onClear: this.clearSelection });
+    }
+    return (
+      <Box flex={false}>
+        <Button
+          tabIndex="-1"
+          hoverIndicator="background-contrast"
+          onClick={this.clearSelection}
+        >
+          <Box
+            background="light-4"
```

Comment:
I'm thinking we should provide more modularity to this background color. If the user is using a dark mode I'm not sure this color would work. maybe it should be fetched from the theme?